pkg/printers/internalversion/printers_test.go

```diff
@@ -2309,7 +2309,7 @@ func TestPrintService(t *testing.T) {
 					ClusterIP: "10.9.8.7",
 				},
 			},
-			"test2\t10.9.8.7\t<none>\t8888:9999/tcp\t<unknown>\n",
+			"test2\tNodePort\t10.9.8.7\t<nodes>\t8888:9999/tcp\t<unknown>\n",
```

Comment:
Why is it not `<none>` but `<nodes>`? I don't think `<nodes>` is a right and meaningful output here.